src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -350,11 +354,14 @@ subroutine init_atm_static(mesh, dims, configs)
        write(mminlu,'(a)') 'USGS'
     case('MODIFIED_IGBP_MODIS_NOAH')
        write(mminlu,'(a)') 'MODIFIED_IGBP_MODIS_NOAH'
+    case('MODIFIED_IGBP_MODIS_NOAH_15s')
+       write(mminlu,'(a)') 'MODIFIED_IGBP_MODIS_NOAH_15s'
```

Comment:
In case the resolution of the input land use data (30s versus 15s) makes no difference to the model, would it be reasonable to set `mminlu` equal to `MODIFIED_IGBP_MODIS_NOAH` if `config_landuse_data` is either `MODIFIED_IGBP_MODIS_NOAH` or `MODIFIED_IGBP_MODIS_NOAH_15s`? That would remove the need for changes to the logic in `mpas_atmphys_landuse.F`.

Reply:
@mgduda agree that mminlu could be set to MODIFIED_IGBP_MODIS_NOAH. Thank you!